contracts/Mars.sol

```diff
@@ -17,6 +18,9 @@ contract Mars is IbcReceiver, Ownable {
 
     string[] supportedVersions = ['1.0', '2.0'];
 
+    /// This function is called for plain Ether transfers, i.e. for every call with empty calldata.
+    receive() external payable {}
```

Comment:
Will this be required for all dapps that want to support fees then?

Reply:
Yes, otherwise it won't be able to receive refunds. I added a comment to clarify that an empty function body is sufficient.

I think I'm gonna add a register payee to Dispatcher so that Dapp can delegate refunds to a different account, either EOA or contract.